### ISSUE TITLE
demo-data-cleaner Exit code returned:', 8

### ISSUE THREAD

**Issue by SivaMaplelabs** [vmware/hillview#429]
root@hillview-root:~/DEPLOY-HILLVIEW/hillview/bin# ./demo-data-cleaner.sh
~/DEPLOY-HILLVIEW/hillview/data/ontime ~/DEPLOY-HILLVIEW/hillview/bin
wget -q https://transtats.bts.gov/PREZIP/On_Time_Reporting_Carrier_On_Time_Performance_1987_present_1987_1.zip
('Exit code returned:', 8)

Looks like access is denied to https://transtats.bts.gov/PREZIP.

**Comment by mbudiu-vmw** [comment]
No, it looks like the government website https://transtats.bts.gov/DL_SelectFields.asp has screwed up its own file naming scheme. I will try to find out what it really is. The download button on their own website does not work anymore if you request a year other than 2018.

**Comment by mbudiu-vmw** [comment]
At this point there is nothing I can do. Maybe someone at the website will notice and fix the form to generate the correct links. Until then we can only download 2018 data.

**Comment by SivaMaplelabs** [comment]
Ok.
